From 359b22d3f775afa33cca9e4f8fb57eadd0ec4118 Mon Sep 17 00:00:00 2001
From: Nico Weber <thakis@chromium.org>
Date: Fri, 23 Apr 2021 16:22:19 +0000
Subject: [PATCH] Make dom distiller protoc plugin explicitly call py2.7

With this, chrome builds with when `/usr/bin/env python` is py3.

Bug: 1202134
Change-Id: Ibbd97a1311ccb34d46f266912c871fd0522f9535
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2848445
Commit-Queue: Nico Weber <thakis@chromium.org>
Commit-Queue: Yaron Friedman <yfriedman@chromium.org>
Commit-Queue: Dirk Pranke <dpranke@google.com>
Auto-Submit: Nico Weber <thakis@chromium.org>
Reviewed-by: Yaron Friedman <yfriedman@chromium.org>
Reviewed-by: Dirk Pranke <dpranke@google.com>
Cr-Commit-Position: refs/heads/master@{#875702}
---
 .../dom_distiller_js/protoc_plugins/json_values_converter.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/third_party/dom_distiller_js/protoc_plugins/json_values_converter.py b/third_party/dom_distiller_js/protoc_plugins/json_values_converter.py
index e86a88c759fcb..cae1a998c4650 100755
--- a/third_party/dom_distiller_js/protoc_plugins/json_values_converter.py
+++ b/third_party/dom_distiller_js/protoc_plugins/json_values_converter.py
@@ -1,7 +1,10 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 # Copyright 2016 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
+#
+# TODO(crbug.com/1202134): Switch run line back to just "python"
+# once things are py3-compatible.
 
 """protoc plugin to create C++ reader/writer for JSON-encoded protobufs
 
